### ISSUE TITLE
Getting "Der Ziel-Link konnte nicht gefunden werden"

### ISSUE THREAD

**Issue by jaschaurbach** [pfefferle/wordpress-webmention#454]
Hi!

Out of the blue I get the error "Der Ziel-Link konnte nicht gefunden werden" from bird.gy - see [here](https://brid.gy/log?module=background&start_time=1704297014&key=agdicmlkLWd5cjULEghSZXNwb25zZSIndGFnOm9oYWkuc29jaWFsLDIwMTM6MTExNjgxNDUzMTA5MDIwMzU5DA)

When I try it to add it manually via https://jascha.wtf/wp-json/webmention/1.0/endpoint I get the same error:

<img width="1000" alt="Bildschirmfoto 2024-01-03 um 16 59 48" src="https://github.com/pfefferle/wordpress-webmention/assets/2017105/9c066271-a8a1-435f-89af-b38e0c7cce70">

<img width="1000" alt="Bildschirmfoto 2024-01-03 um 16 59 54" src="https://github.com/pfefferle/wordpress-webmention/assets/2017105/0609caee-ae61-487c-82ff-de62011b2716">

Any idea what I could do to further debug this issue? The destination of course [exists](https://jascha.wtf/angels-at-chaos-about-volunteering-and-fitting-in/)

**Comment by pfefferle** [comment]
The destination exists, but the plugin can't find a link to your post in the brid.gy html.

/cc @snarfed

**Comment by pfefferle** [comment]
Perhaps we need to change the error message a little to make this clearer!

**Comment by jaschaurbach** [comment]
But it was an answer to a comment. So it might be in the side of Brid.gy to name the parent comment

**Comment by pfefferle** [comment]
Oh is this a threaded comment thing? An answer to a comment to your post, not your post directly?

If so, then this is not supported by the Webmention plugin (yet)!

**Comment by pfefferle** [comment]
The plugin checks for URLs with the same domain and rejects any other request. When we started working on Webmentions we have not thought about bridges to other social network protocols like ActivityPub.

So to support that we would have to think about how to implement that properly.

**Comment by dshanske** [comment]
> The plugin checks for URLs with the same domain and rejects any other request. When we started working on Webmentions we have not thought about bridges to other social network protocols like ActivityPub.
> 
> So to support that we would have to think about how to implement that properly.

I thought Bridgy was supposed  to convert the syndicated copy of a post to their website link for that reason. Maybe I'm confused about that. @snarfed

**Comment by jaschaurbach** [comment]
I jfound out that the parent comment is empty: `681444048580159\\/111681453109020359","comment_parent":"","comment_autho`

**Comment by snarfed** [comment]
> I thought Bridgy was supposed to convert the syndicated copy of a post to their website link for that reason. Maybe I'm confused about that. @snarfed

Right. The root problem here, or at least one, is that https://brid.gy/comment/mastodon/@jascha@ohai.social/111681444048580159/111681453109020359 should have `u-in-reply-to` link pointing to https://jascha.wtf/angels-at-chaos-about-volunteering-and-fitting-in/ , but doesn't.

That post is both a direct reply to https://chaos.social/@jascha@ohai.social/111681444071267594 and an indirect reply to the root post https://chaos.social/@jascha@ohai.social/111679915834398325 . The original post https://jascha.wtf/angels-at-chaos-about-volunteering-and-fitting-in/ is in the latter. Bridgy should still find it, but my guess is that the immediate parent is overriding it somehow. Maybe related to https://github.com/snarfed/bridgy/issues/1617 . I'll add these details there.

Thanks for the report, all!
